### ISSUE TITLE
get_flights/get_airlines not working anymore 

### ISSUE THREAD

**Issue by bartkramer** [supercoderz/pyflightdata#9]
Error from get_flights
```
>>> f.get_flights('kl-klm')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python27\lib\site-packages\pyflightdata\flightdata.py", line 190, in get_flights
    return self._fr24.get_airline_flight_data(url)
  File "C:\Python27\lib\site-packages\pyflightdata\common_fr24.py", line 284, in get_airline_flight_data
    data = self.get_raw_airline_flight_data(url)
  File "C:\Python27\lib\site-packages\pyflightdata\common_fr24.py", line 264, in get_raw_airline_flight_data
    return self.get_raw_data(url, 'tbl-datatable', 'tbody', 'tr') or []
  File "C:\Python27\lib\site-packages\pyflightdata\common.py", line 90, in get_raw_data
    return self.traverse(soup, elemid, elements)
  File "C:\Python27\lib\site-packages\pyflightdata\common.py", line 81, in traverse
    res = res.find(element)
AttributeError: 'NoneType' object has no attribute 'find'
```
Error from get_airlines.
```
f.get_airlines()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python27\lib\site-packages\pyflightdata\flightdata.py", line 148, in get_airlines
    return self._fr24.get_airlines_data(url)
  File "C:\Python27\lib\site-packages\pyflightdata\common_fr24.py", line 211, in get_airlines_data
    data = self.get_raw_airlines_data(url)
  File "C:\Python27\lib\site-packages\pyflightdata\common_fr24.py", line 171, in get_raw_airlines_data
    return self.get_raw_data(url, 'tbl-datatable', 'tbody', 'tr') or []
  File "C:\Python27\lib\site-packages\pyflightdata\common.py", line 90, in get_raw_data
    return self.traverse(soup, elemid, elements)
  File "C:\Python27\lib\site-packages\pyflightdata\common.py", line 81, in traverse
    res = res.find(element)
AttributeError: 'NoneType' object has no attribute 'find'
```

On v0.3. Rest of the functions is working well.

**Comment by E3V3A** [comment]
@bartkramer  Can you post the JSON result for that query?

**Comment by hari-allamraju** [comment]
Are you using the latest version ?

Thanks,
Hari

**Comment by bartkramer** [comment]
Yes I am on the latest version available from pip. 

@E3V3A I can't get any results, it just goes straight to the error.

**Comment by hari-allamraju** [comment]
Can you show us a snipped of your full code? Get flights has been having some issues as fr24 page comes up blank

Get airlines should work fine

Thanks,
Hari

**Comment by bartkramer** [comment]
```
from pyflightdata import FlightData
api=FlightData(email,password)
api.login(email,password)
api.get_flights('kl-klm') OR api.get_airlines()
```
My account is a gold account, I've also tried not signing in and I get the same error in the end.

**Comment by hari-allamraju** [comment]
Fix is ready; expect a new release in the next 24 hours or so - am trying to try and get in another known issue with get_flights fixed

**Comment by bartkramer** [comment]
Super, thanks!

**Comment by hari-allamraju** [comment]
Fixed and released version 0.5.8
@E3V3A this release also fixes the issue with get flights which was failing earlier

In addition the project is now MIT licenced
